docs/design/autoware-architecture/map/map-requirements/overview.md

```diff
@@ -0,0 +1,83 @@
+# Vector Map creation requirement specifications
```

Comment:
Currently, this page is displayed as a subpage under Map requirements folder in the menu. 
Instead, maybe we could consider 
* create a folder called `vector-map-requirements-overview`
* put all the newly created markdown files under that folder
* rename this file to index.md 

This way, we can have a page called "Vector map requirements overview" with pull down menu for category pages.

Before:
![image](https://github.com/autowarefoundation/autoware-documentation/assets/43976834/64d39bdc-59c7-48a4-ae70-a64d861d1de7)

After:
![image](https://github.com/autowarefoundation/autoware-documentation/assets/43976834/fef50ce4-3fac-4165-af5c-cd37806f4cd4)
![image](https://github.com/autowarefoundation/autoware-documentation/assets/43976834/08ca2a8b-e728-4ac0-a6a7-5bfdd66aa5fe)

Reply:
Thank you for your suggestion. I changed files and folders structure in https://github.com/autowarefoundation/autoware-documentation/pull/581/commits/a463059f0e18ee949cf967ecd040659e4bc00315 .